scene/3d/path_3d.cpp

```diff
@@ -93,24 +93,63 @@ void Path3D::_update_debug_mesh() {
 		return;
 	}
 
-	Vector<Vector3> vertex_array;
+	real_t interval = 0.1;
+	const real_t length = curve->get_baked_length();
 
-	for (int i = 1; i < curve->get_point_count(); i++) {
-		Vector3 line_end = curve->get_point_position(i);
-		Vector3 line_start = curve->get_point_position(i - 1);
-		vertex_array.push_back(line_start);
-		vertex_array.push_back(line_end);
+	if (length <= CMP_EPSILON) {
+		RS::get_singleton()->instance_set_visible(debug_instance, false);
+		return;
 	}
 
-	Array mesh_array;
-	mesh_array.resize(Mesh::ARRAY_MAX);
-	mesh_array[Mesh::ARRAY_VERTEX] = vertex_array;
+	const int sample_count = int(length / interval) + 2;
+	interval = length / (sample_count - 1);
+
+	Vector<Vector3> ribbon;
+	ribbon.resize(sample_count);
+	Vector3 *ribbon_ptr = ribbon.ptrw();
+
+	Vector<Vector3> bones;
+	bones.resize(sample_count * 4);
+	Vector3 *bones_ptr = bones.ptrw();
+
+	for (int i = 0; i < sample_count; i++) {
+		const Transform3D r = curve->sample_baked_with_rotation(i * interval, true, true);
+
+		const Vector3 p1 = r.origin;
+		const Vector3 side = r.basis.get_column(0);
+		const Vector3 up = r.basis.get_column(1);
+		const Vector3 forward = r.basis.get_column(2);
+
+		// Path3D as a ribbon.
+		ribbon_ptr[i] = p1;
+
+		// Fish Bone.
```

Comment:
Very nitpicky, not necessarily worth updating the PR just for this. But if other review comments require more changes, could be done alongside.
```suggestion
		// Fish bone.
```